bench/qemu/optimized/block_qcow2-refcount.c.ll

```diff
@@ -749,30 +754,35 @@ do.body208:                                       ; preds = %for.body202, %for.c
 if.then211:                                       ; preds = %do.body208
   %36 = load ptr, ptr %35, align 8
   call void @bdrv_debug_event(ptr noundef %36, i32 noundef 28) #17
-  %.pre172 = load ptr, ptr %file, align 8
+  %.pre173 = load ptr, ptr %file, align 8
   br label %do.end215
 
 do.end215:                                        ; preds = %do.body208, %if.then211
-  %37 = phi ptr [ null, %do.body208 ], [ %.pre172, %if.then211 ]
-  %call219 = call i32 @bdrv_pwrite_sync(ptr noundef %37, i64 noundef %add86185192, i64 noundef %mul, ptr noundef nonnull %call45, i32 noundef 0) #17
+  %37 = phi ptr [ null, %do.body208 ], [ %.pre173, %if.then211 ]
+  %call219 = call i32 @bdrv_pwrite_sync(ptr noundef %37, i64 noundef %add86186193, i64 noundef %mul, ptr noundef nonnull %call45, i32 noundef 0) #17
   %cmp220 = icmp slt i32 %call219, 0
   br i1 %cmp220, label %fail, label %for.cond224.preheader
 
 for.cond224.preheader:                            ; preds = %do.end215
-  br i1 %cmp200149, label %for.body227, label %for.end232
+  br i1 %cmp200149.not, label %for.end232, label %for.body227.preheader
 
-for.body227:                                      ; preds = %for.cond224.preheader, %for.body227
-  %indvars.iv167 = phi i64 [ %indvars.iv.next168, %for.body227 ], [ 0, %for.cond224.preheader ]
+for.body227.preheader:                            ; preds = %for.cond224.preheader
+  %smax170 = call i32 @llvm.smax.i32(i32 %conv9, i32 1)
+  %wide.trip.count171 = zext nneg i32 %smax170 to i64
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597406008